components/prism-uorazor.js

```diff
@@ -0,0 +1,47 @@
+Prism.languages.uorazor = {
+	'comment-hash': {
+		pattern: /#.*/,
+		alias: 'comment',
+		greedy: true
+	},
+	'comment-slash': {
+		pattern: /\/\/.*/,
+		alias: 'comment',
+		greedy: true
+	},
+	'source-layers': {
+		pattern: /\b(?:arms|backpack|blue|bracelet|cancel|clear|cloak|criminal|earrings|enemy|facialhair|friend|friendly|gloves|gray|grey|ground|hair|head|innerlegs|innertorso|innocent|lefthand|middletorso|murderer|neck|nonfriendly|onehandedsecondary|outerlegs|outertorso|pants|red|righthand|ring|self|shirt|shoes|talisman|waist)\b/i,
+		alias: 'function'
+	},
+	'source-commands': {
+		pattern: /\b(?:alliance|attack|cast|clearall|clearignore|clearjournal|clearlist|clearsysmsg|createlist|createtimer|dclick|dclicktype|dclickvar|dress|dressconfig|drop|droprelloc|emote|getlabel|guild|gumpclose|gumpresponse|hotkey|ignore|lasttarget|lift|lifttype|menu|menuresponse|msg|org|organize|organizer|overhead|pause|poplist|potion|promptresponse|pushlist|removelist|removetimer|rename|restock|say|scav|scavenger|script|setability|setlasttarget|setskill|settimer|setvar|sysmsg|target|targetloc|targetrelloc|targettype|undress|unignore|unsetvar|useobject|useonce|useskill|usetype|virtue|wait|waitforgump|waitformenu|waitforprompt|waitforstat|waitforsysmsg|waitfortarget|walk|wfsysmsg|wft|whisper|yell)\b/,
+		alias: 'function'
+	},
+	'tag-name': {
+		pattern: /(^\{%-?\s*)\w+/,
+		lookbehind: true,
+		alias: 'keyword'
+	},
+	'delimiter': {
+		pattern: /^\{[{%]-?|-?[%}]\}$/,
+		alias: 'punctuation'
+	},
+	'string': {
```

Comment:
`string` also has to be greedy. E.g. `say '#something'` is currently highlighted incorrectly.

Also, please move `string` up so it comes right after `comment-slash`. This won't change the language and improves performance.

Reply:
Appreciate you reviewing and leaving clear instructions!  This has been moved/marked as greedy.